TheJumpCloud/support
# restore ids

## Issues
* [SA-3025](https://jumpcloud.atlassian.net/browse/SA-3025) - Restore ID object on Get-JCSystem/ User

## What does this solve?

At one point before the 2.0.0 changeover we began stripping out the .id field on Systems/ Users. The ._id field remains unaffected but some customers like to write their scripts using the .id field. 

## Is there anything particularly tricky?

## How should this be tested?

Pull this branch, run `get-JCSystem` & `get-JCUser`, the .id parameter should be included in the results for all systems/users or individual systems/users

## Screenshots


[SA-3025]: https://jumpcloud.atlassian.net/browse/SA-3025?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SA-3025]: https://jumpcloud.atlassian.net/browse/SA-3025?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
